sdk/spring/compatibility-test.yml

```diff
@@ -0,0 +1,33 @@
+trigger: none
+pr: none
+
+variables:
+  - template: ../../eng/pipelines/templates/variables/globals.yml
+
+stages:
+  - stage:
+    jobs:
+      - job:
+
+        pool:
+          vmImage: "ubuntu-latest"
+
+        steps:
+          - script: |
+              pip install termcolor
+            displayName: 'python module install'
+          - script: |
+              python ./sdk/spring/scripts/compatibility_add_dependencymanagement.py -b 2.6.6 -c 2021.0.1
+            displayName: 'add dependency management'
+          - script: |
+              python ./sdk/spring/scripts/compatibility_deleted_version.py
+            displayName: 'delete version control'
```

Comment:
I guess you mean delete content like `<version>...</version>` for dependencies. It's not `delete version control`. Could you please give a better name?